patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch

```diff
@@ -381,6 +463,81 @@
  
    std::vector<std::string> GetOnSecurityEventProviders();
  
+--- a/chrome/browser/enterprise/signals/signals_utils.cc
++++ b/chrome/browser/enterprise/signals/signals_utils.cc
+@@ -10,7 +10,6 @@
+ #include "components/policy/content/policy_blocklist_service.h"
+ #include "components/policy/core/browser/url_blocklist_manager.h"
+ #include "components/prefs/pref_service.h"
+-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
```

Comment:
The removal of `components/safe_browsing/core/common/safe_browsing_prefs.h` should be dealt with in `remove-unused-preferences-fields.patch`.